spec/jobs/fetch_hearing_locations_for_veterans_job_spec.rb

```diff
@@ -254,7 +254,22 @@
               receive(:record_geomatched_appeal).with("error")
             )
           end
-          expect(Raven).to receive(:capture_exception)
+          expect(Raven).to receive(:capture_exception).with(anything,
+                                                            hash_including(extra: { actionable: false,
+                                                                                    appeal_external_id: anything }))
+
+          subject.perform
+        end
+      end
+
+      context "when non-actionable error is thrown" do
+        let(:nonactionable_error) { Caseflow::Error::VaDotGovMissingFacilityError.new(message: "test", code: 500) }
+        before { allow_any_instance_of(GeomatchService).to receive(:geomatch).and_raise(nonactionable_error) }
+
+        it "records a geomatch error with actionable attribute = false" do
+          expect(Raven).to receive(:capture_exception).with(nonactionable_error,
+                                                            hash_including(extra: { actionable: true,
```

Comment:
the context says 
> records a geomatch error with actionable attribute = false 

But the test is looking for actionable attribute = true
```suggestion
                                                            hash_including(extra: { actionable: false,
```

Reply:
Well at least the context was correct. Thanks!